objects/src/accounts/account_id.rs

```diff
@@ -300,3 +348,18 @@ pub fn validate_account_seed(account: &Account, seed: Word) -> Result<(), Accoun
 
     Ok(())
 }
+
+/// Given a [Digest] returns its proof-of-work.
+pub fn digest_pow(digest: Digest) -> u32 {
+    digest.as_elements()[3].as_int().trailing_zeros()
+}
+
+/// Given a [Digest] returns its hex representaiton.
+pub fn digest_hex(digest: Digest) -> String {
+    to_hex(&digest.as_bytes()).expect("hex formatting failed")
+}
+
+/// Given a [Word] returns its hex representation.
+pub fn word_hex(word: Word) -> String {
+    to_hex(FieldElement::elements_as_bytes(&word)).expect("hex formatting failed")
+}
```

Comment:
We could add `#[cfg(feature = "log")]` attribute to these and then we won't need to make them public.

Reply:
changed